pkg/query/source.go

```diff
@@ -62,27 +71,27 @@ func (s *SourceQueryBuilder) Execute(ctx context.Context, dbExecutor db.Executor
 	}
 }
 
-func (s *SourceQueryBuilder) Results(dbExecutor db.Executor, verboseSearch SearchVerbosity) error {
+func (s *SourceQueryBuilder) Results(w io.Writer, dbExecutor db.Executor, verboseSearch SearchVerbosity) error {
```

Comment:
Very nice addition

Reply:
As you said, "should be easier to add tests now)".